CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ for Rust libraries in [RFC #1105](https://github.com/rust-lang/rfcs/blob/master/
   those wishing to avoid syntax extensions from `diesel_codegen`. See
   http://docs.diesel.rs/diesel/macro.Queryable!.html for details.
 
+* The `Identifiable!` macro can now be used instead of `#[derive(Identifiable)]` for
+  those wishing to avoid syntax extensions from `diesel_codegen`. See
+  http://docs.diesel.rs/diesel/macro.Identifiable!.html for details.
```

Comment:
Good point. I'll probably give the whole changelog a pass before the next release.